pkgs/by-name/ms/msgraph-cli/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildDotnetModule
```

Comment:
Please format with the new formatter.

Reply:
could you elaborate? I'm a little out of loop

Reply:
I think all you need to do is:
1. rebase on master
2. execute `nix-shell` in nixpkgs
3. run nixfmt <your file>

Reply:
You can ignore the failure on deps.nix